src/metric/definition.go

```diff
@@ -905,10 +906,22 @@ var KubeletSpecs = definition.SpecGroups{
 			{Name: "label.*", ValueFunc: definition.Transform(definition.FromRaw("labels"), kubeletMetric.OneMetricPerLabel), Type: sdkMetric.ATTRIBUTE},
 
 			// computed
-			{Name: "cpuCoresUtilization", ValueFunc: toUtilization("cpuUsedCores", "cpuLimitCores"), Type: sdkMetric.GAUGE},
-			{Name: "requestedCpuCoresUtilization", ValueFunc: toUtilization("cpuUsedCores", "cpuRequestedCores"), Type: sdkMetric.GAUGE},
-			{Name: "memoryUtilization", ValueFunc: toUtilization("memoryUsedBytes", "memoryLimitBytes"), Type: sdkMetric.GAUGE},
-			{Name: "requestedMemoryUtilization", ValueFunc: toUtilization("memoryUsedBytes", "memoryRequestedBytes"), Type: sdkMetric.GAUGE},
+			{
+				Name:      "cpuCoresUtilization",
+				ValueFunc: toUtilization(definition.FromRaw("cpuUsedCores"), definition.FromRaw("cpuLimitCores")),
+				Type:      sdkMetric.GAUGE},
```

Comment:
To be consistent with general styling and other changes.
```suggestion
				Type:      sdkMetric.GAUGE
			},
```